public/src/vue/ListView.vue

```diff
@@ -20,6 +32,35 @@ export default {
     BottomPathObjects,
     ProjectsCarousel,
   },
+  data() {
+    return {
+      search: "",
+    }
+  },
+  computed: {
+    suggestionsList: function() {
+      let workshops = [];
+      this.projects.forEach(project => {
+        workshops.push(project.workshop_type);
+      });
+      return Array.from(new Set(workshops));
+    },
+    projects: function() {
+      if(this.search === "") {
+        return this.$root.projects_that_are_accessible;
+      }
+      console.log("SEARCH PROJECT");
+      let projectsList = [];
+      this.$root.projects_that_are_accessible.forEach(project => {
+        if(project.workshop_type != null) {
+          if(project.workshop_type.toLowerCase().includes(this.search.toLowerCase())) {
+            projectsList.push(project);
+          }
+        }
+      });
+      return projectsList;
+    }
```

Comment:
try:

    projects: function() {
      return this.$root.projects_that_are_accessible.filter(project => (
        project.workshop_type != null
        && project.workshop_type.toLowerCase().includes(this.search.toLowerCase())
      )};
    }